{{ cookiecutter.project_name_dashed }}/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,27 +35,19 @@ Generally, **PRs are only reviewed when ALL BOXES are ticked off and all
 automated checks pass** (use the comment section below if you believe that
 your PR is ready to be merged even though not all boxes were ticked off). -->
 
-- \[ \] My code follows the [contributing guidelines][contributing-guidelines]
-  of this project, including, in particular, with regard to any style guidelines
-- \[ \] The title of my PR complies with the
-  [Conventional Commits specification][conv-commits]; in particular, it clearly
-  indicates that a change is a breaking change
-- \[ \] I acknowledge that all my commits will be squashed into a single commit,
+1. [] My code follows the [contributing guidelines][contributing-guidelines]
+  and I acknowledge that all my commits will be squashed into a single commit
   using the PR title as the commit message
-- \[ \] I have performed a self-review of my own code
-- \[ \] I have commented my code in hard-to-understand areas
-- \[ \] I have updated the user-facing documentation to describe any new or
```

Comment:
I think this is not automated. Actually, please expand this to say:

> [...] updated any relevant user- developer- and or administrator-facing documentation [...]

Reply:
Sorry, I hadn't seen that you merged this with another point. Guess that's a point in case for my argument _not_ to merge individual items. 

So please unmerge and extend this point regarding the user facing documentation as described above.